core/build.gradle.kts

```diff
@@ -45,6 +45,7 @@ dependencies {
   // SquiDB
   implementation(Libs.squidb)
   implementation(Libs.squidb_annotations)
+  implementation("androidx.documentfile:documentfile:1.0.1")
```

Comment:
[nit] For consistency, we should store the name of the library in the `Libs.kt` file.

Reply:
move to libs.kt